app/controllers/TaxYearSummaryController.scala

```diff
@@ -152,15 +153,16 @@ class TaxYearSummaryController @Inject()(authActions: AuthActions,
   private def withTaxYearFinancials(taxYear: Int, isAgent: Boolean)(f: List[TaxYearSummaryChargeItem] => Future[Result])
                                    (implicit user: MtdItUser[_]): Future[Result] = {
 
-    financialDetailsService.getFinancialDetails(taxYear, user.nino) flatMap {
+    val taxYearTyped = TaxYear(taxYear - 1, taxYear)
+    financialDetailsService.getFinancialDetails(taxYearTyped, taxYearTyped, user.nino) flatMap {
```

Comment:
looks a bit strange with respect to taxYear we are passing, looks like we are passing the same value

Reply:
We only want the data for that tax year, so the range is the start of taxYearTyped to the end of taxYearTyped, so we pass the same one twice. I could make a new method in financialDetailsService to just get financial details for a single year, if that makes it clearer

Reply:
Have added getFinancialDetailsSingleYear